### PR TITLE
Adds new helper, DisplayTimeText

### PR DESCRIPTION
**Pull request by ShizCalev**
Closes #30969

Noticed that there was a fair bit of duplicated math to calculate how much time was remaining on some items for chat messages and timers.

This cleans up that math by adding a new helper, DisplayTimeText(). 

It takes a value in deciseconds and returns the amount as a text value with the correct measurement proceeding it (days, hours, minutes, seconds.)

This makes it so you will no longer see messages such as "This will take around 1 minutes", instead saying "This will take 60 seconds." Something that says "This will take 1 seconds" now only states "This will take 1 second."

It will roll over to the larger figure once you pass the exact point for the corresponding measurement, IE; 1.1 minutes = 1.1 minutes. 1 hour exact = 60 minutes. 1.1 hours = 1.1 hours, so on so forth.

60ahjhz

Also added defines for second/minute/hour values in deciseconds.

### CODE REVIEW

File: code/__DEFINES/misc.dm [hunk 1]
```diff
@@ -20,6 +20,12 @@ Will print: "/mob/living/carbon/human/death" (you can optionally embed it in a s
 
 #define MIDNIGHT_ROLLOVER		864000	//number of deciseconds in a day
 
+#define DECISECONDS_SECOND		10
+#define DECISECONDS_MINUTE		600
+#define DECISECONDS_HOUR		36000
+#define DECISECONDS_DAY			864000
+#define DECISECONDS_WEEK		6048000
```
Comment (Jordie0608) [inline code/__DEFINES/misc.dm]: This one is unused and could be done without.

**Review by Jordie0608 — CHANGES_REQUESTED** [review]

**Review by Jordie0608 — DISMISSED** [review]

File: code/__HELPERS/time.dm [hunk 2]
```diff
@@ -50,3 +50,16 @@ GLOBAL_VAR_INIT(rollovercheck_last_timeofday, 0)
 			return 5
 		else
 			return 1
+
+//Takes a value of time in deciseconds.
+//Returns a text value of that number in hours, minutes, or seconds.
+/proc/DisplayTimeText(var/time_value)
```
Comment (Cyberboss) [inline code/__HELPERS/time.dm]: remove var/

**Review by Cyberboss — DISMISSED** [review]

**Review by Cyberboss — CHANGES_REQUESTED** [review]
What I said

**Comment by ChangelingRain** [comment]
I'd kind of prefer `1 hour and 6 minutes` instead of `1.1 hours`

**Comment by ShizCalev** [comment]
@ChangelingRain Got it working; I'll make another PR for it in a bit so folks can decide.

**Comment by Jordie0608** [comment]
I think it's fair to say the alternate PR is all around better as it reads akin to normal formatting and avoids uglies like `3.9 minutes`

**Pull request closed without merging**